nuxeo-features/nuxeo-platform-oauth/src/test/java/org/nuxeo/ecm/platform/oauth/tests/OAuth2ChallengeFixture.java

```diff
@@ -169,61 +179,115 @@ public void authorizationShouldValidateRedirectURI() {
         params.put("client_id", "nuxeo-mobile-app");
         params.put("redirect_uri", "nuxeo://authorize");
 
-        cr = responseFromAuthorizationWith(params);
+        cr = responseFromGetAuthorizationWith(params);
         assertEquals(200, cr.getStatus());
 
         // Valid: starting with http://localhost with localhost not part of the domain name
         registerClient("Localhost", "localhost", "", "http://localhost:8080/nuxeo");
         params.put("client_id", "localhost");
         params.put("redirect_uri", "http://localhost:8080/nuxeo");
 
-        cr = responseFromAuthorizationWith(params);
+        cr = responseFromGetAuthorizationWith(params);
         assertEquals(200, cr.getStatus());
 
         // Valid: starting with https
         registerClient("Secure", "secure", "", REDIRECT_URI);
         params.put("client_id", "secure");
         params.put("redirect_uri", REDIRECT_URI);
 
-        cr = responseFromAuthorizationWith(params);
+        cr = responseFromGetAuthorizationWith(params);
         assertEquals(200, cr.getStatus());
     }
 
     @Test
-    public void tokenShouldCreateAndRefreshWithDummyAuthorization() throws IOException {
-        AuthorizationRequest request = new TestAuthorizationRequest(CLIENT_ID, RESPONSE_TYPE, null, REDIRECT_URI,
-                new Date());
-        TestAuthorizationRequest.getRequests().put("fake", request);
+    public void shouldDenyAccess() {
```

Comment:
Maybe add a comment to say that the access is expected to be denied because of the missing `grant_access` parameter